### DISCUSSION TITLE
Issue with GitHub Copilot in IntelliJ Plugin After Waking Up Windows Laptop from Sleep

### DISCUSSION THREAD

**Discussion by zbigniew-r** [community/community#128119] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hello everyone,  

I've been encountering an issue with the GitHub Copilot plugin in IntelliJ IDEA 2024.1.2 on my Windows laptop. The problem arises when I wake up my laptop after it has been put into sleep mode.  
In this situation, I can write a message in the GitHub Copilot Chat, but I don't receive any response. Instead, I see the message: "No steps completed successfully."  The issue gets resolved when I restart IntelliJ IDEA, but it's inconvenient to have to restart the IDE every time I wake up my laptop from sleep.  
I've ensured that my internet connection is stable and that there are no updates pending for either IntelliJ or the GitHub Copilot plugin.  Has anyone else experienced this issue? Are there any known solutions or workarounds that don't involve restarting IntelliJ IDEA? Any help or suggestions would be greatly appreciated.  Thank you in advance for your assistance.

**Comment by AistisTaraskevicius** [comment]
Same, this does appear to be quite recent as I havent experienced this before, though havent been using co-pilot for long.

**Comment by alefranzoni** [comment]
Same here, I'm on **Macbook M1** and after waking up the device from sleep or, sometimes, after unlocking the laptop (without entering into sleep mode) the Chat gets stuck until I restart the IDE. It's a very annoying issue...

**Comment by Juraj-Masiar** [comment]
Same here, using WebStorm on Windows 11. After waking up from sleep, the Chat won't work until I restart whole IDE.

**Comment by LLUMOAI** [comment]
Every time I need to reinstall the IntelliJ plugin to make it work.

**Comment by pgeadas** [comment]
Totally useless plugin right now. Even if you reinstall, it happens again next time your computer comes from sleeping.

**Reply by pgeadas** [reply]
Please check your VPN when starting IntelliJ. It fixed it for me. The VPN needs to be off (or already connected) when IntelliJ starts.

**Comment by davidbmadsen** [comment]
I'm having the same issue with Copilot (ver. 1.5.9.5799) for Rider. I haven't found any other workaround than to restart the IDE whenever it's been left open during sleep mode.  

I assume this happens because there is no re-authentication mechanism to GH upon losing network connection or similar. This is a major annoyance and poses a significant workflow disruption as you constantly have to restart the IDE whenever the computer wakes from sleep mode or there is a network change. Even a prompt to re-authenticate would be a significant quality of life improvement.

**Comment by omasseau** [comment]
Same issue here. This is really annoying.

**Comment by zainengineer** [comment]
Solution is here, you need to update the plugin
https://github.com/orgs/community/discussions/125026#discussioncomment-9702284

**Reply by AistisTaraskevicius** [reply]
Not related and been updating daily.

**Comment by Frosty515** [comment]
Also happens on Arch Linux with CLion on waking from suspend. Might be an upstream issue though as I've noticed it in Visual Studio Code as well.

**Comment by wwzeng1** [comment]
If you'd like to use a better plugin, we built [Sweep AI](https://docs.sweep.dev) using native Java Swing components and regularly profile memory usage and fix startup bugs - it's the most solid option today.

I'd recommend you give it a try!
